src/js/edu-benefits/components/benefits-eligibility/BenefitsSelectionFields.jsx

```diff
@@ -20,11 +22,31 @@ export default class BenefitsSelectionFields extends React.Component {
               options={relinquishableBenefits}
               value={this.props.data.benefitsRelinquished}
               onValueChange={(update) => {this.props.onStateChange('benefitsRelinquished', update);}}/>
+          {showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)
+            ? <DateInput required={showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)}
+                errorMessage="Please provide a response"
+                validation={validateIfDirtyDateObj(this.props.data.benefitsRelinquishedDate, isValidDateField)}
+                label="Effective date"
```

Comment:
Should we have a more descriptive label than just "Effective date"? Like "Effective date of relinquished benefit" or something? @cweber @goldenmeanie